manifests/run.pp

```diff
@@ -161,8 +161,16 @@
         $init_template = 'docker/etc/init.d/docker-run.erb'
         $deprecated_initscript = "/etc/init/${service_prefix}${sanitised_title}.conf"
         $hasstatus  = true
-        $uses_systemd = false
         $mode = '0755'
+
+        case $::lsbdistcodename {
+          'jessie': {
+            $uses_systemd = true
+          }
+          default: {
+            $uses_systemd = false
```

Comment:
Vielleicht ist es sinnvoll die Logik hier zu drehen weil "not uses systemd" jetzt schon die Ausnahme ist. Sonst müssen wir das anpassen wenn Debian 9 stable wird.